src/test/java/io/managed/services/test/billing/BillingModelTest.java

```diff
@@ -95,7 +95,7 @@ public void testAutomaticallyPickSingleCloudAccount() {
             kafka = KafkaMgmtApiUtils.createKafkaInstance(kafkaMgmtApi, payload);
             log.debug(kafka);
             assertEquals(kafka.getBillingModel(), "marketplace");
-            assertEquals(kafka.getMarketplace(), "aws");
```

Comment:
Do not make this change,  this would make a mismatch. Marketplace is part of billing, not what type of instance you want to create, and therefore in this test should stay as "aws"